build/android/BUILD.gn

```diff
@@ -113,6 +113,92 @@ copy("brave") {
   outputs = [ brave_android_output ]
 }
 
+java_strings_grd("android_playlist_strings_grd") {
+  grd_file = "//brave/android/java/org/chromium/chrome/browser/playlist/android_playlist_strings.grd"
+  outputs = [
+    "values/android_playlist_strings.xml",
```

Comment:
these targets don't appear to list every file out like this in other places. They normally looks something like this

```
java_strings_grd("java_strings_grd") {
  defines = chrome_grit_defines
  grd_file = "java/strings/android_touch_to_fill_strings.grd"
  outputs =
      [ "values/android_touch_to_fill_strings.xml" ] +
      process_file_template(
          android_bundle_locales_as_resources,
          [ "values-{{source_name_part}}/android_touch_to_fill_strings.xml" ])
}
```

is there a a reason we have listed them individually? Also should they be under playlist/strings?

Reply:
i took https://github.com/brave/brave-core/blob/024560f295cbb86d02ebc2bed9f0c05fe2450418/build/android/BUILD.gn#L116 as the ref. it's updated as per the suggestion.